src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,5 +1,7 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
```

Comment:
This code section is a Javadoc of the login() method and should be right above it, class fields on the other hand should be at the beginning of the class

Reply:
![image](https://github.com/mate-academy/jv-oop-lesson-0/assets/91253664/9abd2ab9-84e5-4830-9826-898d02ab00ec)
maven wants me to add  [EmptyLineSeparator]